workerapi/client/cli.go

```diff
@@ -1,36 +1,30 @@
 package client
 
 import (
-	"fmt"
-	"log"
-
 	"github.com/apache/thrift/lib/go/thrift"
-	"github.com/scootdev/scoot/workerapi/gen-go/worker"
 	"github.com/spf13/cobra"
 )
 
-type Client struct {
-	rootCmd          *cobra.Command
-	addr             string // modified by flag parsing
-	transportFactory thrift.TTransportFactory
-	protocolFactory  thrift.TProtocolFactory
-	client           *worker.WorkerClient
+type cliClient struct {
+	client
```

Comment:
And then have this not be incorporating, but just have client be of type *client...

That will take a little more typing in commands.go, but really not much. c.Dial() will return a client.

(Note: this separation is why the daemon/client is split up into conn vs cli... Maybe worth looking at)